api/.docker/db/Dockerfile.local

```diff
@@ -3,6 +3,7 @@ FROM postgres:9.5
 
```

Comment:
do we need this file in the repo? Isn't it specific to every dev's local env?

Reply:
This is already there in the repo. I was having some problems with my laptop where it says `something is using port 5432` and I was not able to build the database container locally. So, pushan helped me change the logic here, the new approach is that the port value is read from `.env.local` otherwise it defaults to `5432`